README.md

```diff
@@ -14,9 +14,12 @@ git clone git@github.com:WebexSamples/webex-meetings-quick-start.git
 ### Install Dependencies
 
 ```bash
-npm install
+yarn install
 ```
 
+> [!NOTE]
+> You need to be on Node V18
```

Comment:
Could we add a `.npmrc` file and specify the `engines` parameter in the `package.json` file?

https://stackoverflow.com/a/29349773

Reply:
Added `.npmrc` as requested. Kindly check @adamweeks